### PR TITLE
Add initial management page (fixes #171)

### LINKED ISSUE #171: Make entry points for separate modules for management and payments

### PR DESCRIPTION
**Pull request by muffinresearch**
management

Notes:
* Webpack conf now generates a commons.js file which is the js shared between the payments and management interface.
* Styles are just a starting point - will defintely need updating to spec in due course.
* Commons.js needs minification conf (I'll file a bug for that)

### CODE REVIEW

File: public/js/apps/management/app.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+var Management = React.createClass({
+
+  displayName: 'ManagementApp',
+
+  render: function() {
+    return (
+
+      <div>
+        <header className="top-nav">
+          <h1 className="logo">Firefox Payments</h1>
```
Comment (kumar303) [inline public/js/apps/management/app.jsx]: hmm, should this be localized? I'm not sure

File: public/js/apps/management/app.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          <button>{gettext('Sign Out')}</button>
+        </header>
+
+        <div className="content">
+          <div className="user">
+            <p>Hello, placeholder@placeholder.com</p>
```
Comment (kumar303) [inline public/js/apps/management/app.jsx]: this needs localization. We could fix it up later when we add a real email though.

File: public/js/apps/management/main.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var $ = require('jquery');
+var App = require('./app');
+
+// Common ajax settings.
+$.ajaxSetup({
+  dataType: 'json',
```
Comment (kumar303) [inline public/js/apps/management/main.js]: hmm, it's possible that the [header config](https://github.com/mozilla/payments-ui/blob/master/public/js/actions/user.jsx#L25) might erase this setting.

File: public/js/apps/payment/main.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 var $ = require('jquery');
-var App = require('app');
 var tracking = require('tracking');
+var App = require('./app');
```
Comment (kumar303) [inline public/js/apps/payment/main.js]: nice. I feel like we should do relative imports more often

File: public/js/components/card-icon.jsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var classNames = require('classnames');
+var cx = require('classnames');
```
Comment (kumar303) [inline public/js/components/card-icon.jsx]: why cx?

File: public/js/views/management.jsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
+'use strict';
+
```
Comment (kumar303) [inline public/js/views/management.jsx]: what is this file used for? It seems like ``management/app.jsx`` replaced it, no?

**Comment by kumar303** [comment]
r+wc

File: public/js/apps/management/app.jsx (hunk 2 above)
Comment (muffinresearch) [inline public/js/apps/management/app.jsx]: Yeah, that's what I was thinking too given there's no format func yet.

File: public/js/components/card-icon.jsx (hunk 5 above)
Comment (muffinresearch) [inline public/js/components/card-icon.jsx]: It's a convention in react stuff.

File: public/js/views/management.jsx (hunk 6 above)
Comment (muffinresearch) [inline public/js/views/management.jsx]: Oh yeah - this should be deleted.

File: public/js/apps/management/app.jsx (hunk 2 above)
Comment (kumar303) [inline public/js/apps/management/app.jsx]: oh bummer, I thought gettext already had formatting.

File: public/js/components/card-icon.jsx (hunk 5 above)
Comment (kumar303) [inline public/js/components/card-icon.jsx]: wfm

File: public/js/apps/management/main.js (hunk 3 above)
Comment (muffinresearch) [inline public/js/apps/management/main.js]: Yep looks like it. I'll file a bug.

**Pull request merged**